api/tests/acceptance/application/certification-center-controller_test.js

```diff
@@ -218,20 +218,21 @@ describe('Acceptance | API | Certification Center', () => {
     });
   });
 
-  describe('GET /api/certification-centers/{id}/students', () => {let request;
+  describe('GET /api/certification-centers/{certificationCenterId}/sessions/{sessionId}/students', () => {
```

Comment:
Ici j'ai l'impression qu'on a trop de tests d'acceptance, on test un peu la combinatoire. 
D'après moi, je pense qu'il faut avoir un test "happy path" qui montre que les tuyaux sont bien branchés (on renvoie une 200 + `deep equal` sur le JSON de retour par exemple).

Les autres tests peuvent être testés différemment avec des boucles de feedback plus rapides et des set-up plus léger : 
- _**should return Forbidden Access when the certificationCenter does not exist**_
T'as un test du usecase qui montre qu'une exception `ForbiddenAccess` est bien throwée si le centre n'existe pas, un test d'intégration qui montre que la méthode de répo fonctionne correctement et (si c'est pas déjà le cas) tu peux ajouter un TU dans `certification-center_index_.js` (voir `session_index_test.js`) qui s'assure qu'une erreur `ForbiddenAccess` est bien transformée dans le code http que tu veux. 

- **_should return Forbidden Access when user are not register in the certificationCenter_**
idem

- **_when user is not connected_**
Faire un TU dans dans `certification-center_index_.js` (voir `session_index_test.js`) qui vérifie ça.